src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md

```diff
@@ -5,7 +5,8 @@ CHANGELOG
 -----
 
  * allowed to pass an optional `LoggerInterface $logger` instance to the `Router`
-
+ * Passing a `RouterInterface` without `WarmableInterface` is deprecated.
```

Comment:
Using a `RouterInterface` that does not implement the `WarmableInterface` is deprecated and will not be supported in Symfony 5.0.

Reply:
Oh, and this must then be added to `UPGRADE-4.1.md` and `UPGRADE-5.0.md` (with a slight rewording of course) too.